### ISSUE TITLE
Add test for PATCH on json-ld doc

### ISSUE THREAD

**Issue by michielbdejong** [solid/test-suite#148]
The PATCH tests work against a fixture that is a [turtle doc with a single triple](https://github.com/solid-contrib/solid-crud-tests/blob/85d5608/test/surface/update.test.ts#L239).

We could add a test that tests PATCH against a fixture that is a JSON-LD doc, to test what https://github.com/nodeSolidServer/node-solid-server/pull/1695 fixes.

**Comment by melvincarvalho** [comment]
Good idea.  All the different representations (turtle, json-ld, (x)html) should have the same functionality.

**Comment by TallTed** [comment]
Additionally, I think that `PATCH` on a document containing a single triple is very different than `PATCH` on a document containing multiple triples, including but not limited to documents that use such Turtle features as the `;` or `,` syntactic sugar and/or comments and/or syntactically uninterpreted but humanly assistive whitespace, and that a broad if not exhaustive range of these should be tested to ensure both that a) the functionality works as intended based on the specification and b) that the implementer and/or user can best understand that intention.
